Userland/Libraries/LibWasm/AbstractMachine/Interpreter.cpp

```diff
@@ -368,6 +368,34 @@ ALWAYS_INLINE static T rotr(T value, R shift)
     return (value >> shift) | (value << ((-shift) & mask));
 }
 
+template<typename T>
+ALWAYS_INLINE static i32 clz(T value)
+{
+    if (value == 0)
+        return sizeof(T) * CHAR_BIT;
+
+    if constexpr (sizeof(T) == 32)
+        return __builtin_clz(value);
+    else if constexpr (sizeof(T) == 64)
+        return __builtin_clzll(value);
+    else
+        VERIFY_NOT_REACHED();
+}
+
+template<typename T>
+ALWAYS_INLINE static i32 ctz(T value)
+{
+    if (value == 0)
+        return sizeof(T) * CHAR_BIT;
+
+    if constexpr (sizeof(T) == 32)
```

Comment:
Same, `sizeof(i32)` is 4, not 32.

(the build will probably fail because of this)